scripts/snyk-test.js

```diff
@@ -26,7 +26,10 @@ async function snykTest(cwd) {
           '--dev',
           `--json-file-output=${tmpPath}`,
         ],
-        { cwd }
+        {
+          cwd,
+          maxBuffer: 1024 * 1024 * 5, // 5 MB (default is 1)
```

Comment:
[nit] We could as well raise this to be 50 MB :) Or use `spawn` with I/O passed through instead of `execFile`

Reply:
i thought about both: 
- using spawn and forwarding stdio, we silenced the output recently cause it kept confusing folks about what are the actual vulnerabilities that needs attention
- 50 MB that we can do yeah, any reason why we may want to have a smaller limit instead?

Reply:
> any reason why we may want to have a smaller limit instead?

Not really, I don't think 50 MB is something where we'd need to worry about memory exhaustion or anything like that